tests/testthat/test-Rules-methods.R

```diff
@@ -344,31 +344,37 @@ test_that("nextBest-NextBestMinDist returns expected values of the objects", {
   nb_md <- NextBestMinDist(target = 0.3)
 
   result <- nextBest(nb_md, 50, samples, model, data)
-  expect_identical(result, list(value = 50))
+  expect_identical(result$value, 50)
+  expect_snapshot(result$probs)
+  vdiffr::expect_doppelganger("Plot of nextBest-NextBestMinDist", result$plot)
 })
 
-test_that("nextBest-NextBestMinDist returns expected values of the objects (no doselimit)", {
-  data <- h_get_data(placebo = FALSE)
+test_that("nextBest-NextBestMinDist returns expected values and plot (with placebo)", {
+  data <- h_get_data(placebo = TRUE)
   model <- h_get_logistic_log_normal()
   samples <- h_as_samples(
-    list(alpha0 = c(-1.8, -3.8, -2.2, -1.6), alpha1 = c(1.7, 3.3, 5.1, 2.2))
+    list(alpha0 = c(-0.38, -0.13, 1.43, 2.57), alpha1 = c(1.67, 1.3, 1.77, 2.51))
   )
-  nb_md <- NextBestMinDist(target = 0.3)
+  nb_prob_mtd <- NextBestMinDist(target = 0.1)
```

Comment:
```suggestion
  nb_md <- NextBestMinDist(target = 0.1)
```
just for consistency with other parts of the code.